CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - Unreleased
```

Comment:
Why the Sem Ver breaking change if it's just additions? Is it the protected_attributes part? If it's not necessary to be breaking, that would be good. Or if it is, that's worth a bullet here.

Reply:
There was a slight typo in the headers, but under the `Removed` section there's the note that support for `protected_attributes` was removed.  Do you think removing support for a rails gem deserves more reasoning around why it results in breaking changes and a major version bump?

Reply:
I'm not certain if removing that should be a breaking change. Was it part of this gem's interface...or just part of its implementation.

If you don't mind chasing the version bump, the major is fine. But it will need to be chased, since our Sem Ver requirements will disqualify `bundle update` or `dependabot`-driven updates.

Reply:
It was implementation, but because of the implementation, it will require changes on the usage side if mass assignment was currently used for attributes created by `large_text_field`.

Reply:
I'm okay with chasing this through since we're going to be walking all the way up the stack anyway for the rails5 upgrade.